WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.swift

```diff
@@ -252,15 +270,15 @@ private extension CardPresentPaymentsModalViewController {
 
 // MARK: - Actions
 private extension CardPresentPaymentsModalViewController {
-    func didTapPrimaryButton() {
+    @objc func didTapPrimaryButton() {
```

Comment:
Was adding @objc needed because of the touchUpInside changes above?

Reply:
Yes, exactly.